index.html

```diff
@@ -35,7 +35,7 @@
                         <a href="about.html" title="Over ons">Over Ons</a>
                     </div>
                     <div class="menu-item menu-button">
-                        <a href="nutrition.html" title="Voeding">Voeding</a>
+                        <a href="voeding.html" title="Voeding">Voeding</a>

```

Comment:
Ik had de namen in het Engels gedaan omdat dat in het technisch ontwerp staat. De URL's worden straks aangepast met een RewriteRule.